packages/manager/modules/pci/src/projects/project/storages/objects/object/delete/delete.routing.js

```diff
@@ -1,21 +1,28 @@
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('pci.projects.project.storages.objects.delete', {
-    url: '/delete?containerId',
+    url: '/delete?containerId?isHighPerfStorage',
```

Comment:
```suggestion
    url: '/delete?containerId&isHighPerfStorage',
```

Reply:
Not sure this belongs in the url. If customer wants a direct access to this they shouldn't be bothering if their container is `highPerfStorage` or not

Reply:
Yes that would be the normal case. But right now to get high performance storage, we have to pass a static region to the API to get the details of this. I dont see any other way of doing this.
Also to combine both swift and open IO containers in the same list page, we are fetching results(openIO) based on static region and passing `isHighPerfStorage` as a parameter from the 2API.